app/internal/page/certificate_provider_details.go

```diff
@@ -88,25 +83,20 @@ var mobileRegex = regexp.MustCompile(`^(?:07|\+?447)\d{9}$`)
 func (d *certificateProviderDetailsForm) Validate() validation.List {
```

Comment:
Can't comment on the actual line as its not changed but I think we can drop `mobileRegex` now its centralised in validation.